assets/styles/style.css

```diff
@@ -0,0 +1,155 @@
+* {
+    padding: 0;
+    margin: 0;
+}
+
+.container {
+    background-image: url("/assets/img/top-section-back.svg");
+    background-size: 100%;
+    min-height: 100vh;
+} 
+
+.head {
+    display: flex;
+    padding: 51px 104px 0;
+}
+
+.head-menu {
+    display: flex;
+    margin-left: auto;
+    margin-right: 0;
+}
+
+.menu {
+    color: #FFFFFF;
+    margin-top: 6px;
+    font-family: Roboto;
+    font-size: 24px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 28px;
+    letter-spacing: 0em;
+    text-align: center;
+}
+
+a {
+    text-decoration: none;
+    color: #FFFFFF;
+}
+
+/* a:active {
+    border-bottom: 4px solid #F5BD41;
+} 
+--> as i think active link with line 
+*/ 
```

Comment:
look towards :hover. I would recommend using it for  Stays and Attractions links